yul-format/CMakeLists.txt

```diff
@@ -0,0 +1,2 @@
+add_executable(yul-format main.cpp)
```

Comment:
Shouldn't this go into the same directory where also yulrun and yulopti are?

Reply:
I considered that. But it's not a test tool, it's actually a tool that can potentially be very useful for anyone outside that would want to (auto-)pretty-print their code. VS for example uses clang-format executable to reformat your code while you are typing. I had similar in mind with Yul here. I am open for "contra" however. :)

Reply:
Same is true for yulopti.

Reply:
Does this mean we could create a /tools/ directory where we can put such tools (not necessarily restricted to test purposes)?